### PR TITLE
MGMT-2947 - move day2 ignition from general to per node ignition

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

File: internal/bminventory/inventory_test.go [hunk 1]
```diff
@@ -1670,8 +1670,8 @@ var _ = Describe("cluster", func() {
 					mockClusterApi.EXPECT().VerifyClusterUpdatability(gomock.Any()).Return(nil).Times(1)
 				})
 				It("update api vip dnsname success", func() {
-					fileName := fmt.Sprintf("%s/worker.ign", clusterID)
-					mockS3Client.EXPECT().Upload(gomock.Any(), gomock.Any(), fileName).Return(nil).Times(1)
+					// fileName := fmt.Sprintf("%s/worker.ign", clusterID)
```
Comment (danielerez) [inline internal/bminventory/inventory_test.go:1673]: these lines can be removed now

File: internal/bminventory/inventory_test.go [hunk 2]
```diff
@@ -3917,29 +3919,13 @@ var _ = Describe("Register AddHostsCluster test", func() {
 				OpenshiftVersion: &openshiftVersion,
 			},
 		}
-		fileName := fmt.Sprintf("%s/worker.ign", clusterID)
-		mockS3Client.EXPECT().Upload(gomock.Any(), gomock.Any(), fileName).Return(nil).Times(1)
+		//fileName := fmt.Sprintf("%s/worker.ign", clusterID)
```
Comment (danielerez) [inline internal/bminventory/inventory_test.go:3922]: can be removed

File: internal/bminventory/inventory_test.go (hunk 1 above)
Comment (yevgeny-shnaidman) [reply internal/bminventory/inventory_test.go]: fixed

File: internal/bminventory/inventory_test.go (hunk 2 above)
Comment (yevgeny-shnaidman) [reply internal/bminventory/inventory_test.go]: fixed

**Pull request merged**